src/main/java/mekanism/api/gas/Gas.java

```diff
@@ -268,7 +268,13 @@ public Fluid getFluid() {
     public Gas registerFluid(String name) {
         if (fluid == null) {
             if (FluidRegistry.getFluid(name) == null) {
-                fluid = new Fluid(name, getIcon(), getIcon(), getTint());
+                int tint = getTint();
+                //Fluids use ARGB so make sure that we are not using a fully transparent tint.
+                // This fixes issues with some mods rendering our fluids as invisible
+                if ((tint >> 24 & 0xFF) == 0) {
+                    tint = 0xFF << 24 | tint;
```

Comment:
Why the bitshifting? just use `0xFF000000` for OR & AND?

Reply:
Will change it tomorrow, I mainly copied the bitshifting for checking alpha from how blood magic was calculating the alpha and my mind was blanking for combining values.